### PR TITLE
Adding a "funny" caption 🥇to the Luke Skywalker slide.

### PR DESCRIPTION
**Pull request by silver-io**
The slide wasn't funny enough so I've changed it.

### CODE REVIEW

**Comment by DiMarieDiMarie** [comment]
I think this all looks fantastic! Let's see that meme on the big screen!

**Review by DiMarieDiMarie — APPROVED** [review]

**Comment by githubteacher** [comment]
⭐️

**Comment by DiMarieDiMarie** [comment]
💯

File: _posts/0000-01-04-silver-io.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
...
 layout: slide
-title: "Welcome to our slide deck!"
+title: "Javier's Slide!"
 ---
 
 ![luke](https://cloud.githubusercontent.com/assets/16547949/25400950/3a263b30-29c2-11e7-8f09-02127726c1e1.jpg)
 {: .center}
 
-Caption here
+I can't let go!
```
Comment (DiMarieDiMarie) [inline _posts/0000-01-04-silver-io.md:9]: 💯 💯 Excellent work, Javier! 💯 💯

Comment (githubteacher) [inline _posts/0000-01-04-silver-io.md:9]: This is "funny".

Comment (DiMarieDiMarie) [inline _posts/0000-01-04-silver-io.md:9]: Great!

File: _posts/0000-01-04-silver-io.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "Javier's Slide!"
```
Comment (githubteacher) [inline _posts/0000-01-04-silver-io.md:3]: 🎉 🌮

**Review by githubteacher — APPROVED** [review]
@silver-io This is great work. Really good job.

**Pull request merged**